spec/controllers/works/drafts_spec.rb

```diff
@@ -0,0 +1,123 @@
+# frozen_string_literal: true
+require "spec_helper"
+
+describe WorksController do
+  include LoginMacros
+
+  let!(:drafts_user_pseud) { create(:pseud, name: "Visiting User Pseud") }
```

Comment:
Not super important, but this and the next `let` don't need the `!`, since they'll get invoked in the `before` block.